cmd/stringer/golden_test.go

```diff
@@ -397,6 +401,80 @@ func (i Token) String() string {
 }
 `
 
+const toLower_in = `type Day int
+const (
+	Monday Day = iota
+	Tuesday
+	Wednesday
+	Thursday
+	Friday
+	Saturday
+	Sunday
+)
+`
+
+const toLower_out = `func _() {
```

Comment:
warning: don't use underscores in Go names; const toLower_out should be toLowerOut (golint)